bear-language-requirements.yaml

```diff
@@ -0,0 +1,411 @@
+ApertiumLintBear:
```

Comment:
Please run YAMLLintBear on this file, since YAML is guided by indentation, I see inconsistent styles being used for the bears with no dependencies i.e. with `[]` in comparison to the rest of the file, which can be avoided easily

Reply:
The changes have been made by @sladyn98. Marking as resolved.